Error: A `label` alone cannot be used to provide an accessible label for a group of fields. Labels can only be associated with a single field using the `for` attribute. Use `fieldset` and `legend`.
Error: Use `aria-describedby` to reference a group description.
Error: For usability, it is recommended to put the group description at the top of the component, just after the label.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
diff --git a/output/Checkbox Group with Accessible Description/1/26b072ba-c186-497f-9a9b-855a92aa662a.html b/output/Checkbox Group with Accessible Description/1/26b072ba-c186-497f-9a9b-855a92aa662a.html
index 728fd21..31bfd55 100644
--- a/output/Checkbox Group with Accessible Description/1/26b072ba-c186-497f-9a9b-855a92aa662a.html	
+++ b/output/Checkbox Group with Accessible Description/1/26b072ba-c186-497f-9a9b-855a92aa662a.html	
@@ -1,14 +1,16 @@
-<label>What fruits do you like?</label>
-<div>
-  <input type="checkbox" id="bananas" name="fruits" value="bananas">
-  <label for="bananas">Bananas</label>
-</div>
-<div>
-  <input type="checkbox" id="oranges" name="fruits" value="oranges">
-  <label for="oranges">Oranges</label>
-</div>
-<div>
-  <input type="checkbox" id="apples" name="fruits" value="apples">
-  <label for="apples">Apples</label>
-</div>
-<small>You can change this later</small>
\ No newline at end of file
+<fieldset aria-describedby="fruit-desc">
+  <legend>What fruits do you like?</legend>
+  <small id="fruit-desc">You can change this later</small>
+  <div>
+    <input type="checkbox" id="bananas" name="fruits[]" value="bananas">
+    <label for="bananas">Bananas</label>
+  </div>
+  <div>
+    <input type="checkbox" id="oranges" name="fruits[]" value="oranges">
+    <label for="oranges">Oranges</label>
+  </div>
+  <div>
+    <input type="checkbox" id="apples" name="fruits[]" value="apples">
+    <label for="apples">Apples</label>
+  </div>
+</fieldset>
\ No newline at end of file
